### DISCUSSION TITLE
How do I set up authentication in Flyte?

### DISCUSSION THREAD

**Discussion by SandraGH5** [flyteorg/flyte#1072] [Q&A] [answered]

**Comment by SandraGH5** [comment, marked as answer]
Please visit "Authentication in Flyte" at https://docs.flyte.org/projects/cookbook/en/latest/auto/deployment/cluster/auth_setup.html.

**Reply by nicklofaso** [reply]
@SandraGH5 This link seems to be broken. I believe this is the new link?
https://docs.flyte.org/en/latest/deployment/cluster_config/auth_setup.html
